nucypher/blockchain/eth/actors.py

```diff
@@ -1544,6 +1547,19 @@ def block_until_ready(self, poll_rate: int = None, timeout: int = None):
                 self.nickname, self.nickname_metadata = nickname_from_seed(self.checksum_address)
                 break
 
+            # Provide periodic feedback to the user
+            if not bonded or not funded:
+                now = maya.now()
+                delta = now - last_provided_feedback
+                if delta.total_seconds() >= feedback_rate:
+                    if not bonded and not funded:
+                        waiting_for = "bonding and funding"
+                    else:
+                        waiting_for = "bonding" if not bonded else "funding"
+                    emitter.message(f"    ⓘ Worker not fully started - still waiting for {waiting_for} ...",
```

Comment:
```suggestion
                     emitter.message(f"    ⓘ Worker is not qualified - still waiting for {waiting_for} ...",
```

Just playing with the language a bit - let me know what you think, or accept! :-)